mcs/class/corlib/corlib.dll.sources

```diff
@@ -1758,8 +1758,7 @@ corert/RuntimeAugments.cs
 ../../../external/corefx/src/System.Memory/src/System/Runtime/InteropServices/*.cs
 
 ../../../external/corefx/src/Common/src/CoreLib/System/Buffers/Text/FormattingHelpers.CountDigits.cs
-../../../external/corefx/src/System.Memory/src/System/Buffers/StandardFormat.cs
-../../../external/corefx/src/System.Memory/src/System/Buffers/OperationStatus.cs
+../../../external/corefx/src/System.Memory/src/System/Buffers/*.cs
```

Comment:
We don't want to include all the types as they are old implementation. What is weird though is why it didn't fail and why there is no public API change

Reply:
The reason is that we have this line in master already at line 1756 😄

Reply:
Looks like you added it in https://github.com/mono/mono/commit/e1e94cb6b183096c06cfe80add57e6cd4bffe339